test/alternator/test_ttl.py

```diff
@@ -57,7 +58,7 @@ def passes_or_raises(expected_exception, match=None):
 # very slow on Scylla or reasonably fast depends on the
 # alternator_ttl_period_in_seconds configuration - test/alternator/run sets
 # it very low, but Scylla may have been run manually.
-@pytest.fixture(scope="session")
+@pytest.fixture(scope=testpy_test_fixture_scope)
```

Comment:
Also these fixtures should probably be "module".

Reply:
changed